### ISSUE TITLE
Get test/assessment data to partner

### ISSUE THREAD

**Issue by PatReynolds** [FreeUKGen/Coordination#537]

**Comment by PatReynolds** [comment]
Emailed trustees 7/11/2017

**Comment by PatReynolds** [comment]
Lemon can only do 100% - are trustees happy with that, or can someone do a section?

**Comment by richpomfret** [comment]
This includes transcriber info/profiles - so we need a different version. Further discussions with Barrie on this and while we figure out a way for Monimine to connect (possible through new dev server) he will prepare another export of the data and Lemon will provide DB schema.

**Comment by richpomfret** [comment]
Shall request new/updated data file export from Barrie and then put this on hold until test facility is finalised. Once the latter is ready, we can give any agreed party access to a jailed version for potential exploration or api creation.
